api/src/pcapi/core/bookings/repository.py

```diff
@@ -585,6 +585,7 @@ def _get_filtered_booking_pro(
             User.email.label("beneficiaryEmail"),
             User.phoneNumber.label("beneficiaryPhoneNumber"),
             Stock.beginningDatetime.label("stockBeginningDatetime"),
+            Stock.id.label("stockId"),
```

Comment:
(détail) accès direct : 

```suggestion
            Booking.stockId,
```

(ça ne change pas grand chose, c'est juste plus évident de le récupérer comme ça)